### PR TITLE
[🇹🇷 - 📝] Translation issue fixed.

### PR DESCRIPTION
**Pull request by eren885**

### CODE REVIEW

File: Langs/1055.lang [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-Donate= Bağış
+Donate=Bağış
 Check for Updates=Güncellemeleri kontrol et
 Your Windows 11 Compatibility Results are Below=Windows 11 ile uyumluluk sonuçlarınız aşağıdadır
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=WhyNotWin11'e https://www.whynotwin11.org/ ile ulaş!
-Results Based on Currently Known Requirements!=Sonuçlar şu an Microsoft'un erdiği bilgilerle derlenmiştir!
-Translation by=Bu uygulama şu kişi tarafından çevirilmiştir;
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=WhyNotWinn11'e şimdi https://www.whynotwin11.org/ üzerinden ulaş!
+Results Based on Currently Known Requirements!=Sonuçlar, şu anki bilinen gereksinlimlere göre derlenmiştir!
+Translation by=Bu uygulama Türkçe'ye şu kişi tarafından çevrildi;
```
Comment (micwoj92) [inline Langs/1055.lang:11]: No need for that Turkish here, I think it is logical.

Comment (micwoj92) [reply Langs/1055.lang:11]: So I think only `Bu uygulama şu kişi tarafından çevrildi;` will be good, right?

Comment (eren885) [reply Langs/1055.lang:11]: yeah

**Comment by micwoj92** [comment]
Thank you for updates!

**Comment by eren885** [comment]
yaa

**Pull request merged**